hack/.ci/set_dependency_version

```diff
@@ -117,6 +117,8 @@ elif name == 'logging':
     names = ['fluent-bit-plugin-installer', 'loki-curator', 'telegraf']
 elif name == 'etcd-custom-image':
     names = ['etcd']
+elif name == 'egress-filter-refresher':
+    names = ['egress-filter-blackholer', 'egress-filter-firewaller']
```

Comment:
These images seem to be rather managed by https://github.com/gardener/gardener-extension-shoot-networking-filter. Do we need such change just because this extension uses this `hack/.ci/set_dependency_version` script?

Reply:
Yes. But duplicating the script alternatively, looks not better.

Reply:
Putting unrelated things under gardener/gardener is not a good option either. in future gardener/gardener can provide only the utility class and each and every extension can have its own version of this script that makes use of the utility class. And extension can maintain its own mapping for such special cases. I don't have capacity right now to invest into this but something like this should be the way forward.

For now let's proceed with this change as anyways even now the script contains components that are not directly deployed by gardener itself (not present in its images.yaml) - for example external-dns-management. https://github.com/gardener/gardener/blob/34d0ba98aa49d2d181648a3acbdaf8c04ff4fef3/hack/.ci/set_dependency_version#L114-L115